### PR TITLE
Add getting started Jupyter Notebook

### PR DESCRIPTION
**Pull request by bertt**
Jupyter Notebook for Getting Started with BruTile

### CODE REVIEW

**Comment by pauldendulk** [comment]
Interesting. So how does it work. Should this be deployed somewhere so anyone can use it? How can I test this myself?

**Comment by bertt** [comment]
see https://www.hanselman.com/blog/AnnouncingNETJupyterNotebooks.aspx

**Comment by xivk** [comment]
I'm keeping an eye on this one! :+1:

**Comment by bertt** [comment]
Online version: https://hub-binder.mybinder.ovh/user/bertt-brutile-a08x93qo/notebooks/brutile_getting_started.ipynb

**Comment by bertt** [comment]
online link: https://mybinder.org/v2/gh/bertt/brutile/patch-1

**Comment by pauldendulk** [comment]
Ik see a file in some kind of folder in that link. I should be able to run something, no?

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,80 @@
+FROM jupyter/scipy-notebook:latest
+
+# Install .NET CLI dependencies
+
+ARG NB_USER=jovyan
```
Comment (pauldendulk) [inline Dockerfile]: Hoezo jovyan?

**Pull request merged**